src/fiatconnect/ReviewScreen.tsx

```diff
@@ -24,27 +29,47 @@ import { createFiatConnectTransfer, refetchQuote } from 'src/fiatconnect/slice'
 import FiatConnectQuote from 'src/fiatExchanges/quotes/FiatConnectQuote'
 import { CICOFlow } from 'src/fiatExchanges/utils'
 import i18n from 'src/i18n'
-import { localCurrencyExchangeRatesSelector } from 'src/localCurrency/selectors'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import {
+  getCurrenciesFromRegionCode,
+  localCurrencyExchangeRatesSelector,
+} from 'src/localCurrency/selectors'
 import { emptyHeader } from 'src/navigator/Headers'
 import { navigate, navigateBack } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
 import { StackParamList } from 'src/navigator/types'
+import { userLocationDataSelector } from 'src/networkInfo/selectors'
 import colors from 'src/styles/colors'
 import fontStyles from 'src/styles/fonts'
 import variables from 'src/styles/variables'
 import { Currency } from 'src/utils/currencies'
 
 type Props = NativeStackScreenProps<StackParamList, Screens.FiatConnectReview>
 
+const disclaimerIsNeeded = (fiatType: FiatType, countryCode: string | null) => {
+  const countryCurrencies = getCurrenciesFromRegionCode(countryCode)
+  for (const countryCurrency of countryCurrencies) {
+    if (fiatType === convertToFiatConnectFiatCurrency(countryCurrency as LocalCurrencyCode)) {
+      return false
+    }
+  }
+  return true
+}
+
 export default function FiatConnectReviewScreen({ route, navigation }: Props) {
   const { t } = useTranslation()
   const dispatch = useDispatch()
   const { flow, normalizedQuote, fiatAccount, shouldRefetchQuote } = route.params
   const fiatConnectQuotesLoading = useSelector(fiatConnectQuotesLoadingSelector)
   const fiatConnectQuotesError = useSelector(fiatConnectQuotesErrorSelector)
+  const { countryCodeAlpha2 } = useSelector(userLocationDataSelector)
   const [showingExpiredQuoteDialog, setShowingExpiredQuoteDialog] = useState(
     normalizedQuote.getGuaranteedUntil() < new Date()
   )
+  const showFeeDisclaimer = useMemo(
+    () => disclaimerIsNeeded(normalizedQuote.getFiatType(), countryCodeAlpha2),
+    [normalizedQuote, countryCodeAlpha2]
+  )
```

Comment:
I think this logic can be pretty drastically simplified by just using the `getDefaultLocalCurrencyCode` selector which is what the app uses to select the users default currency when the create a new account.

I believe this is what charlie meant when he said
> In pseudocode, this is what I'd expect:
if (selectedCurrency !== localCurrency) {
  show disclaimer
}

Something like this
```suggestion
  const defaultLocaleCurrencyCode =  useSelector(getDefaultLocalCurrencyCode)
  const showFeeDisclaimer = useMemo(
    () => normalizedQuote.getFiatType() !== convertToFiatConnectFiatCurrency(defaultLocaleCurrencyCode),
    [normalizedQuote, defaultLocaleCurrencyCode]
  )
```

Reply:
Yeah I originally had implemented it using `getDefaultLocalCurrencyCode` but some countries support multiple currencies, which means that the disclaimer may be shown unnecessarily in those cases.

Reply:
Ahh gotcha, I've updated the code again.